templates/mdnext-blog/pages/_app.js

```diff
@@ -1,6 +1,16 @@
 import { Chakra } from '@components/Chakra';
 import { DefaultSeo } from 'next-seo';
+import getShareImage from '@jlengstorf/get-share-image';
 
+const socialImage = getShareImage({
+  title: 'Nextjs + MDX Starter pack',
+  tagline: 'A test of sorts.',
+  cloudName: process.env.CLOUDINARY_USER_ID,
+  imagePublicID: process.env.CLOUDINARY_IMAGE_ID,
```

Comment:
I think we should probably remove this from being a env variable -- or possibly name is OG_IMAGE_BASE